### PR TITLE
ref: Let functional components call useProjects() & useOrganization() 1/2

### PR DESCRIPTION
**Pull request by ryan953**
Don't call withProjects on a functional component

### CODE REVIEW

File: static/app/views/settings/earlyFeatures/settingsForm.tsx [hunk 1]
```diff
@@ -8,19 +8,20 @@ import JsonForm from 'sentry/components/forms/jsonForm';
 import type {JsonFormObject} from 'sentry/components/forms/types';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import {t} from 'sentry/locale';
-import type {Organization, OrganizationAuthProvider, Scope} from 'sentry/types';
+import type {OrganizationAuthProvider, Scope} from 'sentry/types';
 import {useApiQuery} from 'sentry/utils/queryClient';
-import withOrganization from 'sentry/utils/withOrganization';
+import useOrganization from 'sentry/utils/useOrganization';
```
Comment (evanpurkhiser) [inline static/app/views/settings/earlyFeatures/settingsForm.tsx:13]: looks like some `useOrganization` snuck into the PR too

Comment (ryan953) [reply static/app/views/settings/earlyFeatures/settingsForm.tsx:13]: ya, i did them both when they were around. i can re-title it

Comment (ryan953) [reply static/app/views/settings/earlyFeatures/settingsForm.tsx:13]: the target is the projects

there might also be one or two usePageFilters()

File: static/app/views/settings/earlyFeatures/index.tsx [hunk 2]
```diff
@@ -5,21 +5,15 @@ import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
...
 import EarlyFeaturesSettingsForm from 'sentry/views/settings/earlyFeatures/settingsForm';
 import PermissionAlert from 'sentry/views/settings/organization/permissionAlert';
 
-type Props = {
-  organization: Organization;
-  projects: Project[];
-} & RouteComponentProps<{}, {}>;
-
-function OrganizationGeneralSettings(props: Props) {
+export default function OrganizationGeneralSettings(props: RouteComponentProps<{}, {}>) {
```
Comment (michellewzhang) [inline static/app/views/settings/earlyFeatures/index.tsx:13]: interesting, this file didn't even need `projects`

File: static/app/views/settings/earlyFeatures/settingsForm.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/settings/earlyFeatures/settingsForm.tsx:13]: oh, more of the txn stuff used that. so stay tuned for that last pr

**Review by michellewzhang — APPROVED** [review]

**Pull request merged**